packages/rpc-rx/src/cached.spec.js

```diff
@@ -75,17 +75,4 @@ describe('createCachedObservable', () => {
       observable2
     ).not.toEqual(observable1);
   });
-
-  it('functions as an subject', (done) => {
```

Comment:
Not the case anymore - and I don't believe it is used anywhere either, i.e. calling `.next` on the observable